### ISSUE TITLE
Crash on Nexus 6 running an app.

### ISSUE THREAD

**Issue by DaveShuckerow** [flutter/flutter#3277]
## Steps to Reproduce

I was running an app that included a card size animating from small to large, and on nexus 6, this made the view grow from smaller than the screen to larger.  It seems that this crashed the app.

## Flutter Doctor

[✓] Flutter (on Mac OS, channel alpha)
    • Flutter at /Users/djshuckerow/.flutter
    • Framework revision d5deea4 (2 weeks ago), engine revision ce3b454

[✓] Android toolchain - develop for Android devices (Android SDK 23.0.2)
    • Android SDK at /Users/djshuckerow/Library/Android/sdk
    • Platform android-23, build-tools 23.0.2
    • Java(TM) SE Runtime Environment (build 1.8.0_72-b15)

[✓] iOS toolchain - develop for iOS devices (Xcode 7.3)
    • XCode at /Applications/Xcode.app/Contents/Developer
    • Xcode 7.3, Build version 7D175

[✓] Atom - a lightweight development environment for Flutter
    • Atom installed; Flutter plugin version 0.2.1

## Logs and Crash Reports

--------- beginning of system
W/System.err: java.lang.NullPointerException: Attempt to invoke interface method 'java.util.Iterator java.util.List.iterator()' on a null object reference
W/System.err: 	at org.domokit.sky.shell.FlutterSemanticsToAndroidAccessibilityBridge.removePersistentNode(FlutterSemanticsToAndroidAccessibilityBridge.java:261)
W/System.err: 	at org.domokit.sky.shell.FlutterSemanticsToAndroidAccessibilityBridge$PersistentAccessibilityNode.update(FlutterSemanticsToAndroidAccessibilityBridge.java:326)
W/System.err: 	at org.domokit.sky.shell.FlutterSemanticsToAndroidAccessibilityBridge.updateSemanticsNode(FlutterSemanticsToAndroidAccessibilityBridge.java:241)
W/System.err: 	at org.domokit.sky.shell.FlutterSemanticsToAndroidAccessibilityBridge.updateSemanticsTree(FlutterSemanticsToAndroidAccessibilityBridge.java:219)
W/System.err: 	at org.chromium.mojom.semantics.SemanticsListener_Internal$Stub.accept(SemanticsListener_Internal.java:79)
W/System.err: 	at org.chromium.mojo.bindings.RouterImpl.handleIncomingMessage(RouterImpl.java:236)
W/System.err: 	at org.chromium.mojo.bindings.RouterImpl.access$000(RouterImpl.java:18)
W/System.err: 	at org.chromium.mojo.bindings.RouterImpl$HandleIncomingMessageThunk.accept(RouterImpl.java:30)
W/System.err: 	at org.chromium.mojo.bindings.Connector.readAndDispatchMessage(Connector.java:245)
W/System.err: 	at org.chromium.mojo.bindings.Connector.readOutstandingMessages(Connector.java:204)
W/System.err: 	at org.chromium.mojo.bindings.Connector.onAsyncWaiterResult(Connector.java:170)
W/System.err: 	at org.chromium.mojo.bindings.Connector.access$100(Connector.java:25)
W/System.err: 	at org.chromium.mojo.bindings.Connector$AsyncWaiterCallback.onResult(Connector.java:150)
W/System.err: 	at org.chromium.mojo.system.impl.CoreImpl.onAsyncWaitResult(CoreImpl.java:573)
W/System.err: 	at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
W/System.err: 	at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:53)
W/System.err: 	at android.os.Handler.dispatchMessage(Handler.java:102)
W/System.err: 	at android.os.Looper.loop(Looper.java:148)
W/System.err: 	at android.app.ActivityThread.main(ActivityThread.java:5417)
W/System.err: 	at java.lang.reflect.Method.invoke(Native Method)
W/System.err: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
W/System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
F/flutter : [FATAL:jni_android.cc(249)] Check failed: false. Please include Java exception stack in crash report
--------- beginning of crash
F/libc    : Fatal signal 6 (SIGABRT), code -6 in tid 11254 (leafy)
F/DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
F/DEBUG   : Build fingerprint: 'leafy/shamu/shamu:6.0.1/MOB30D/2704746:user/release-keys'
F/DEBUG   : Revision: '0'
F/DEBUG   : ABI: 'arm'
F/DEBUG   : pid: 11254, tid: 11254, name: leafy  >>> com.leafy <<<
F/DEBUG   : signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
F/DEBUG   : Abort message: '[FATAL:jni_android.cc(249)] Check failed: false. Please include Java exception stack in crash report
F/DEBUG   : '
F/DEBUG   :     r0 00000000  r1 00002bf6  r2 00000006  r3 b6f43b7c
F/DEBUG   :     r4 b6f43b84  r5 b6f43b34  r6 0000000b  r7 0000010c
F/DEBUG   :     r8 be997f20  r9 00000000  sl be998088  fp b4d3d728
F/DEBUG   :     ip 00000006  sp be997a80  lr b6cb3b61  pc b6cb5f50  cpsr 400f0010
F/DEBUG   : 
F/DEBUG   : backtrace:
F/DEBUG   :     #00 pc 00041f50  /system/lib/libc.so (tgkill+12)
F/DEBUG   :     #01 pc 0003fb5d  /system/lib/libc.so (pthread_kill+32)
F/DEBUG   :     #02 pc 0001c30f  /system/lib/libc.so (raise+10)
F/DEBUG   :     #03 pc 000194c1  /system/lib/libc.so (__libc_android_abort+34)
F/DEBUG   :     #04 pc 000174ac  /system/lib/libc.so (abort+4)
F/DEBUG   :     #05 pc 000d32c5  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #06 pc 000c1c03  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #07 pc 000b9867  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #08 pc 0008e6f3  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #09 pc 000e279d  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #10 pc 000e210b  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #11 pc 000d37ab  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #12 pc 000c34ff  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #13 pc 000c3999  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #14 pc 000c3d29  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #15 pc 000c4787  /data/app/com.leafy-1/lib/arm/libsky_shell.so
F/DEBUG   :     #16 pc 0025e505  /data/app/com.leafy-1/oat/arm/base.odex (offset 0x21b000)
F/DEBUG   : 
F/DEBUG   : Tombstone written to: /data/tombstones/tombstone_04
W/ActivityManager:   Force finishing activity com.leafy/org.domokit.sky.shell.SkyActivity

> For more information about diagnosing and reporting Flutter bugs, please see [https://flutter.io/bug-reports/](https://flutter.io/bug-reports/).

**Comment by abarth** [comment]
@Hixie Looks like an issue in the accessibility integration.

**Comment by Hixie** [comment]
Dupe of #3177.

**Comment by DaveShuckerow** [comment]
Seems to be caused by lastpass.  I turned lastpass off in accessibility settings and the app ran correctly.

**Comment by Hixie** [comment]
Closing in favour of 3177.
